samtranslator/internal/schema_source/aws_serverless_api.py

```diff
@@ -173,6 +173,7 @@ class EndpointConfiguration(BaseModel):
 TracingEnabled = Optional[PassThroughProp]
 OpenApiVersion = Optional[Union[float, str]]  # TODO: float doesn't exist in documentation
 AlwaysDeploy = Optional[bool]
+IndividualRecordSet = Optional[bool]
```

Comment:
Should be plural?

Reply:
This means the API will create a standalone Route53 RecordSet. So I feel it's not plural.

Reply:
Oh nvm it could be two. You're right.

Reply:
I wonder if `SeparateRecordSets` would be clearer. 🤔 I can't articulate well why "individual" feels slightly off... maybe because it's often used to refer to people? ("an individual")